REST/src/main/java/pl/jakubowskiprzemyslaw/REST/listeners/RabbitListenerService.java

```diff
@@ -0,0 +1,23 @@
+package pl.jakubowskiprzemyslaw.REST.listeners;
+
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.jakubowskiprzemyslaw.REST.models.BoardsViewHandler;
+import pl.jakubowskiprzemyslaw.tajgertim.models.view.BoardsView;
+
+@Service
```

Comment:
Please take a look at Javadoc for this annotation. It mentions Evans' DDD and stateless operation. Most (all?) of your services have some state in, don't they?

Reply:
Links for you to peruse:

1. https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Service.html
2. https://stackoverflow.com/questions/6827752/whats-the-difference-between-component-repository-service-annotations-in
3. https://lostechies.com/jimmybogard/2008/08/21/services-in-domain-driven-design/
4. http://gorodinski.com/blog/2012/04/14/services-in-domain-driven-design-ddd/
5. https://4programmers.net/Forum/Java/273806-roznica_miedzy_@component_@repository_@service_annotations_in_spring